features/models.py

```diff
@@ -25,9 +25,9 @@ class Meta:
 class Feature(TranslatableModel, TimestampedModel):
     source_id = models.CharField(
         verbose_name=_("source identifier"),
-        unique=True,
+        db_index=True,
         max_length=200,
-        help_text=_("Unique identifier indicating the source"),
+        help_text=_("Identifier for the feature in the source"),
```

Comment:
could you elaborate on the use case? 🙂 features that are the same in the source, but different in ahti? what will be the difference then?

Reply:
ah, same id, but different `source_type`s?

Reply:
Indeed, features might have the same `source_id`, but they should have different `source_type` in that case. Which makes me think that I should actually add `unique_together` constraint for `source_type` and `source_id`.

Reply:
yeah, `unique_together` would be nice to have then 🙂